src/test/java/io/axoniq/axonserver/connector/AbstractAxonServerIntegrationTest.java

```diff
@@ -69,6 +68,12 @@ public abstract class AbstractAxonServerIntegrationTest {
                     .withNetworkAliases("axonserver")
                     .waitingFor(Wait.forHttp("/actuator/health").forPort(8024));
 
+    private static String getDockerImageName() {
+        String envVariable = System.getenv("AXON_SERVER_IMAGE");
+        System.out.println("ENV VARIABLE: [" + envVariable + "]");
```

Comment:
We should remove this System.out which I have introduced to validate if the environment variable is found by GitHub once the `axon-server-api` has a snapshot present with DCB support.

Reply:
We should!